### PR TITLE
TG-87: User Employee SignUp

### PR DESCRIPTION
**Pull request by lucianofrailelopez**
Creation of the form and validations, just check if the validation and visualization of the form is well done, the sending of data will be done in another ticket.

### CODE REVIEW

File: src/Components/SignUp/index.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+          register={register}
+          error={errors.isProjectManager?.message}
+        />
+        <Input
+          label="Active"
+          name="active"
+          type="checkbox"
+          register={register}
+          error={errors.active?.message}
+        />
```
Comment (Javier-Cervera) [inline src/Components/SignUp/index.js:147]: Users shouldn't be able to select their status or role

**Review by Javier-Cervera — APPROVED** [review]
Looks good! Just a small change

**Review by NicolasFontana — APPROVED** [review]
Looks nice!

**Review by iaracriscenti — APPROVED** [review]
Looks fine!

**Pull request merged**